# Parameters with for the objects used

objectsInfo: {
  nObjects: 8,
  ID1: {
    Filename: prismAV6x6x6,
    Description: Prism 6x6x6,
    nPoses: 1,
    Pose1: [0.050,0.000,0.000,0,89],
  },
  ID2: {
    Filename: prismAV10x8x4,
    Description: Prism 10x8x4,
    nPoses: 2,
    Pose1: [0.060,1.570,0.000,0,179],
    Pose2: [0.070,1.570,1.570,0,179],
  },
  ID3: {
    Filename: prismAV20x6x5,
    Description: Prism 20x6x5,
    nPoses: 3,
    Pose1: [0.040,0.000,0.000,0,179],
    Pose2: [0.050,1.570,0.000,0,179],
    Pose3: [0.120,1.570,1.570,0,179],
  },
  ID4: {
    Filename: bowlAV,
    Description: Bowl,
    nPoses: 1,
    Pose1: [0.015,0.000,0.000,0,1],
  },
  ID5: {
    Filename: cinderBlockAV,
    Description: Cinder Block,
    nPoses: 1,
    Pose1: [0.015,0.000,0.000,0,89],
  },
  ID6: {
    Filename: handleAV,
    Description: Door Handle,
    nPoses: 1,
    Pose1: [0.015,0.000,0.000,0,359],
  },
  ID7: {
    Filename: gasketAV,
    Description: Gasket,
    nPoses: 1,
    Pose1: [0.015,0.000,0.000,0,179],
  },
  ID8: {
    Filename: drillAV,
    Description: Cordless Drill,
    nPoses: 7,
    Pose1: [0.012,0.000,0.000,0,359],
    Pose2: [0.084,1.459,-0.751,0,359],
    Pose3: [0.082,1.398,-0.014,0,359],
    Pose4: [0.040,0.280,-1.040,0,359],
    Pose5: [0.249,2.860,-0.400,0,359],
    Pose6: [0.068,-1.287,-0.477,0,359],
    Pose7: [0.048,-1.570,1.480,0,359],
  },
}
